tests/macsec/test_fault_handling.py

```diff
@@ -91,7 +91,11 @@ def check_new_mka_session():
             ).format(dut_ingress_sa_table_orig, dut_ingress_sa_table_new)
             return True
         assert wait_until(30, 5, 2, check_new_mka_session), (
-            "New MKA session not established within expected time. ")
+            "New MKA session did not establish within expected time."
+            "Current status: '{}'.".format(
+                check_new_mka_session()
```

Comment:
it does have failed reason in this assertion, run check_new_mka_session() again is not a good idea I think. Please remove this line